apps/user/interfaces/user.interface.ts

```diff
@@ -23,14 +23,35 @@ export interface Organisation {
 }
 
 export interface UsersProfile {
-    id?: string;
-    username?: string;
-    email?: string;
-    firstName?: string;
-    lastName?: string;
-    supabaseUserId?: string;
-    userOrgRoles?: object;
-  }
+  id?: string;
+  username?: string;
+  email?: string;
+  firstName?: string;
+  lastName?: string;
+  supabaseUserId?: string;
+  userOrgRoles?: UserOrgRole[];
+}
+
+interface UserOrgRole {
```

Comment:
Let's follow naming convention for interfaces.

Reply:
Resolved